src/QualificationRunner.Core/Services/QualificationRunner.cs

```diff
@@ -486,5 +494,38 @@ public void RemoveByName(JObject obj, string propertyName)
 
          return list;
       }
+
+      private void copySubDirectory(string sourceDir, string destinationDir, bool copyFiles = false)

```

Comment:
use false by default to not copy the files in the ContentFolder. (otherwise all md files will be copied here as well which is done somewhere else)